packages/css-library/src/stylesheets/base/va.scss

```diff
@@ -277,7 +277,11 @@ table {
   @include media($medium-screen) {
     background: $color-gray-lightest;
     .usa-banner-inner {
+      font-size: 12px;
       padding-left: scale-rem(1.5rem);
+      p {
+        font-size: 12px;
+      }
```

Comment:
Changes to these styles are needed to maintain parity with what's currently in production. Leaving font-size out here will render text to 16px.